src/main/scala/org/hammerlab/guacamole/DistributedUtil.scala

```diff
@@ -312,19 +321,24 @@ object DistributedUtil extends Logging {
     skipEmpty: Boolean,
     function: (Pileup, Pileup) => Iterator[T],
     reference: ReferenceGenome): RDD[T] = {
+
+    val referenceBC = reads1.context.broadcast(reference)
```

Comment:
> Does this re-broadcast the reference genome every time we call pileupFlatMapTwoRDDs?

I guess it would, but we currently call this function (or its counterparts) once in each variant caller, and that doesn't seem super likely to change afaict?

This is currently called once in the codebase, by SomaticStandardCaller, and twice in DistributedUtilSuite, so this doesn't seem like a problem to me.

We could make this a wrapper for a version of this method that takes a `Broadcast[ReferenceGenome]` instead of a `ReferenceGenome`; let me know if you prefer that. I think it cleaner to not force that on callers if we don't have to, personally.